keras/models.py

```diff
@@ -740,7 +740,7 @@ def load_weights(self, filepath, by_name=False):
         else:
             layers = self.layers
         if by_name:
-            topology.load_weights_from_hdf5_group_by_name(f, layers)
+            topology.load_weights_from_hdf5_group_by_name(f, layers, skip_mismatch=skip_mismatch)
```

Comment:
Introduce line return after `(` to reduce line length

Reply:
👍

Reply:
I believe I fixed this according to the rest of the code style. Alternatively, I could use the following syntax:
```python
foo(
    bar=0,
    baz=1
)
```

Whichever you prefer.